cmake/configs/base.cmake

```diff
@@ -212,11 +212,18 @@ config_bool(
     DEFAULT OFF
 )
 
+set(default_build_type "Debug")
+if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
+  message(STATUS "Setting default build type to '${default_build_type}'.")
```

Comment:
I think this could be a little clearer - we're always setting the default build type, but in this clause we're "Defaulting build type to ..."